src/Mealz/MealBundle/Controller/ParticipantController.php

```diff
@@ -136,25 +125,15 @@ public function swap(Participant $participant, NotifierInterface $notifier)
 
         $notifier->sendAlert($chefbotMessage);
 
-        // Return JsonResponse
-        $ajaxResponse = new JsonResponse();
-        $ajaxResponse->setData(array(
-            'url' => $this->generateUrl('MealzMealBundle_Participant_unswap', array(
-                'participant' => $participant->getId(),
-            )),
-            'id' => $participant->getId(),
-            'actionText' => $this->get('translator')->trans('swapped', array(), 'action'),
-        ));
-
-        return $ajaxResponse;
+        return $this->generateResponse('MealzMealBundle_Participant_unswap', 'swapped', $participant);
     }
 
     /**
      * @param Participant $participant
```

Comment:
Redundant @param and @return annotation.

Reply:
Done.